projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -597,8 +631,9 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
         return this._emptyRowDimension;
     }
 
+    protected _pivotValueCloneStrategy: IDataCloneStrategy = new DefaultDataCloneStrategy();
     protected override _defaultExpandState = false;
-    protected override _filterStrategy: IFilteringStrategy = new DimensionValuesFilteringStrategy();
+    protected override _filterStrategy: IFilteringStrategy = new DimensionValuesFilteringStrategy(this.pivotValueCloneStrategy);
```

Comment:
Fairly certain this won't refresh properly if the clone strategy is reset, no? Since you have a setter anyway, you might want to refresh this one too or use an alternative approach